Apps/Src/main.c

```diff
@@ -72,7 +74,7 @@ int main(){
 			break;
 		}
 
-		// TODO: Implement heartbeat for RUN light at a visible frequency
+		heartbeat();
```

Comment:
When I compile, I get "warning: implicit declaration of function ‘heartbeat’ [-Wimplicit-function-declaration]". You need to add a function prototype for heartbeat() to the top of main.c, so the compiler knows that this function is defined.